pretty_midi/pretty_midi.py

```diff
@@ -306,20 +306,37 @@ def __get_instrument(program, channel, track, create_new):
                     # Store this as the last note-on location
                     note_on_index = (event.channel, event.note)
                     last_note_on[note_on_index].append((
-                        self.__tick_to_time[event.time],
-                        event.velocity))
+                        event.time, event.velocity))
```

Comment:
Coming from computer graphics, comparing floats by equality scares me. We always had to use epsilon, otherwise nightmares may happen. When we have the original integers at hand it seems to me that it's safer to use them for comparison.